src/shogun/lib/DynamicArray.h

```diff
@@ -611,7 +764,7 @@ template <class T> class CDynamicArray :public CSGObject
 			// current_num_elements as size (see parameter
 			// registration) the cloned version has less memory
 			// allocated than known to dynarray. We fix this here.
-			cloned->m_array.num_elements = cloned->m_array.current_num_elements;
+			// cloned->num_elements = cloned->m_array.size();
```

Comment:
This one the another place may broken serialization. But it will broken SGObject if not comment it. Maybe we need a better way to register parameter.